### PR TITLE
feat(band_icon_tooltip): Hovering band icon now displays tooltip

### PR DESCRIPTION
**Pull request by SylEze**
Tooltip content is "Price oracle powered by Band Protocol"

Solves #521

### CODE REVIEW

File: atomic_defi_design/qml/Components/CexInfoTrigger.qml [hunk 1]
```diff
@@ -5,6 +5,18 @@ import QtQuick.Controls 2.15
...
     onClicked: cex_rates_modal.open()
+    hoverEnabled: true
+
+    DefaultTooltip {
+        visible: mouse_area.containsMouse
+
+        delay: 500
+
+        contentItem: DefaultText {
+            text_value: "Price oracle powered by Band Protocol"
```
Comment (Milerius) [inline atomic_defi_design/qml/Components/CexInfoTrigger.qml:19]: Usage of https://doc.qt.io/qt-5/qtquick-internationalization.html#1-use-qstr-for-all-literal-user-interface-strings is required here i think.

Comment (naezith) [reply atomic_defi_design/qml/Components/CexInfoTrigger.qml:19]: Yes, please use `qsTr(...)` so the text can be translated.

**Review by naezith — COMMENTED** [review]
All good except the `qsTr(...)`.

**Review by smk762 — APPROVED** [review]
sighted in https://github.com/KomodoPlatform/atomicDEX-Desktop/actions/runs/321995166

**Pull request merged**